Userland/Libraries/LibGUI/FileSystemModel.cpp

```diff
@@ -142,12 +138,29 @@ void FileSystemModel::Node::traverse_if_needed()
     }
 }
 
+OwnPtr<FileSystemModel::Node> FileSystemModel::Node::create_child(String const& child_name)
+{
+    String child_path = String::formatted("{}/{}", full_path(), child_name);
```

Comment:
I think this wants to be `LexicalPath::join()`.

Reply:
Indeed. This code actually spawned `LexicalPath::join` but I guess all those changes were mangled together in the original PR.